src/spacel/provision/app/ingress_resource.py

```diff
@@ -102,3 +111,19 @@ def app_sg(self, orbit, region, app, sg_ref='Sg'):
                              orbit.name, region)
                 return None
             raise e
+
+    def app_eips(self, orbit, region, app, eips=[]):
```

Comment:
Yep; that'd work (with `eips = eips or []` in the function body).

Or don't make it a default value: having the non-recursive call provide a `[]` itself would also work.